client/html/src/Client/Html/Account/Profile/Account/Standard.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Aimeos\Client\Html\Account\Profile\Account;
+
+use Illuminate\Support\Facades\Log;
+
+/**
+ * Default implementation of acount profile address HTML client.
+ *
+ * @package Client
+ * @subpackage Html
+ */
+class Standard
+    extends \Aimeos\Client\Html\Common\Client\Summary\Base
+    implements \Aimeos\Client\Html\Common\Client\Factory\Iface
+{
+
+    /** client/html/account/profile/address/subparts
+     * List of HTML sub-clients rendered within the account profile address section
+     *
+     * The output of the frontend is composed of the code generated by the HTML
+     * clients. Each HTML client can consist of serveral (or none) sub-clients
+     * that are responsible for rendering certain sub-parts of the output. The
+     * sub-clients can contain HTML clients themselves and therefore a
+     * hierarchical tree of HTML clients is composed. Each HTML client creates
+     * the output that is placed inside the container of its parent.
+     *
+     * At first, always the HTML code generated by the parent is printed, then
+     * the HTML code of its sub-clients. The address of the HTML sub-clients
+     * determines the address of the output of these sub-clients inside the parent
+     * container. If the configured list of clients is
+     *
+     *  array( "subclient1", "subclient2" )
+     *
+     * you can easily change the address of the output by readdressing the subparts:
+     *
+     *  client/html/<clients>/subparts = array( "subclient1", "subclient2" )
+     *
+     * You can also remove one or more parts if they shouldn't be rendered:
+     *
+     *  client/html/<clients>/subparts = array( "subclient1" )
+     *
+     * As the clients only generates structural HTML, the layout defined via CSS
+     * should support adding, removing or readdressing content by a fluid like
+     * design.
+     *
+     * @param array List of sub-client names
+     * @since 2019.07
+     * @category Developer
+     */
+    private $subPartPath = 'client/html/account/profile/account/subparts';
+    private $subPartNames = [];
+
+    protected function getSubClientNames(): array
+    {
+        return [];
+    }
+
+    public function getSubClient(string $type, string $name = null): \Aimeos\Client\Html\Iface
+    {
+        // TODO: Implement getSubClient() method.
+    }
+
+    /**
+     * Returns the HTML code for insertion into the body.
+     *
+     * @param string $uid Unique identifier for the output if the content is placed more than once on the same page
+     * @return string HTML code
+     */
+    public function getBody(string $uid = ''): string
+    {
+        $view = $this->getView();
+
+        $html = '';
+        foreach( $this->getSubClients() as $subclient ) {
+            $html .= $subclient->setView( $view )->getBody( $uid );
+        }
+        $view->addressBody = $html;
```

Comment:
Replace all occurrences of "address" by "account" everywhere

Reply:
replaced every occurance of "address"